### PR TITLE
Add Gorilla-people as playable species

### PR DESCRIPTION
**Pull request by ghost**
Gorilla meat
gorillameat
Side view
gorillaside
Back view
gorillaback
Front view
gorillafront
Gorilla hide
gorillahide
This gorilla wants the Lizardpeople to recognize the Gorillian Genocide (Over a gorillion were killed).
gorillawithajob
A gorilla doing honest work.
janigorilla
Gorilla warfare.
gorillawarfare

### CODE REVIEW

**Comment by Amelia0010** [comment]
y tho

**Comment by fluffe9911** [comment]
This is cool as heck but would prob be better as a genetic mutation or meltdown result

**Comment by nesisknesis** [comment]
we need this

**Comment by jcatjmeow** [comment]
No

**Comment by nesisknesis** [comment]
do you even play the game

**Comment by Nickvr628** [comment]
https://statici.behindthevoiceactors.com/behindthevoiceactors/_img/chars/gorilla-spongebob-squarepants-88.2.jpg

Also we all know the question we all want to ask, and I hope the answer is yes.

**Comment by Altoids1** [comment]
I'm sure absolutely no one will make any outlandishly racist remarks with these guys around, ever, at all.

File: code/modules/language/gorillian.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/datum/language/gorillian
+	name = "Gorillian"
+	desc = "The common language of upright gorillas, composed of slurred grunts and groans."
```
Comment (Altoids1) [inline code/modules/language/gorillian.dm]: Why not just have them speak Monkey?

**Review by Altoids1 — CHANGES_REQUESTED** [review]
You have like, no modularization, with any of the /tg/ files you're editing. Check out our guide on how to do that, over here:

https://github.com/yogstation13/Yogstation-TG/blob/master/yogstation/README.md

File: config/game_options.txt [hunk 2]
```diff
@@ -441,6 +441,7 @@ ROUNDSTART_RACES human
 
 ## Races that are strictly worse than humans that could probably be turned on without balance concerns
 ROUNDSTART_RACES lizard
+ROUNDSTART_RACES gorilla
```
Comment (Altoids1) [inline config/game_options.txt]: Ehhh....

File: code/modules/language/gorillian.dm (hunk 1 above)
Comment (ghost) [reply code/modules/language/gorillian.dm]: Made them speak monkey, and modularization should be good now.

**Comment by Smudged** [comment]
Well, now I'm invested.
I'll repeat my sentiment I had on the discord, this race HAS potential, but it also has the potential to be MASSIVELY abused.

Now some changes I'd kinndaa like to see to start is
-Gorilla's are vegetarian's they don't like gross stuff, or meat, only fruit and vegetables will sate their hunger.
-More drawbacks, at the moment, they're everything lizard's aren't, and they're only a notch below a human, rather than being multiple notches.

File: code/modules/mob/living/carbon/human/human.dm [hunk 3]
```diff
@@ -1019,7 +1019,10 @@
 
 /mob/living/carbon/human/species/lizard
 	race = /datum/species/lizard
-
+// yogs start
+/mob/living/carbon/human/species/gorilla
+	race = /datum/species/gorilla
+// yogs end
```
Comment (Altoids1) [inline code/modules/mob/living/carbon/human/human.dm]: Can you dump this class definition into ``/yogstation/code/modules/mob/living/carbon/human/human.dm``?

File: code/modules/reagents/reagent_containers/blood_pack.dm [hunk 4]
```diff
@@ -69,7 +69,10 @@
 
 /obj/item/reagent_containers/blood/lizard
 	blood_type = "L"
-
+// yogs start -- gorilla people
+/obj/item/reagent_containers/blood/gorilla
+	blood_type = "G"
+// yogs end
```
Comment (Altoids1) [inline code/modules/reagents/reagent_containers/blood_pack.dm]: Can you dump this class definition into ``/yogstation/code/modules/reagents/reagent_containers/blood_pack.dm``?

File: code/modules/food_and_drinks/food/snacks/meat.dm [hunk 5]
```diff
@@ -86,7 +86,14 @@
...
 	foodtype = MEAT | RAW
-
+// yogs start -- gorilla people
+/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/gorilla
+	icon_state = "gorillameat"
+	desc = "Damn dirty steak."
+	filling_color = "#FF0000"
+	tastes = list("meat" = 4, "fur" = 1)
+	foodtype = MEAT | RAW
+// yogs end
```
Comment (Altoids1) [inline code/modules/food_and_drinks/food/snacks/meat.dm]: Can you dump this class definition into ``/yogstation/code/modules/food_and_drinks/food/snacks/meat.dm``?

**Review by Altoids1 — CHANGES_REQUESTED** [review]

**Comment by Nickvr628** [comment]
At first I was against this but in reality this is amazing and should be merged.

**Comment by fluffe9911** [comment]
Im fluffe9911 and I approve of this pr @Altoids1

**Review by SomeguyManperson — APPROVED** [review]
everything should check out

**Review by fluffe9911 — APPROVED** [review]
Im fluffe9911 and I approve of this pr @Altoids1

**Review by Altoids1 — APPROVED** [review]

**Pull request merged**